master/reference/calicoctl/resources/profile.md

```diff
@@ -59,14 +59,17 @@ spec:
 
 | Field       | Description                 | Accepted Values   | Schema | Default    |
 |-------------|-----------------------------|-------------------|--------|------------|
-| action      | Action to perform when matching this rule. | allow, deny, log | string | | 
+| action      | Action to perform when matching this rule. | allow, deny, log, pass | string | | 
 | protocol    | Positive protocol match.  | tcp, udp, icmp, icmpv6, sctp, udplite, integer 1-255. | string | |
 | notProtocol | Negative protocol match. | tcp, udp, icmp, icmpv6, sctp, udplite, integer 1-255. | string | |
 | icmp        | ICMP match criteria.     | | [ICMP](#icmp) | |
 | notICMP     | Negative match on ICMP. | | [ICMP](#icmp) | |
 | source      | Source match parameters. |  | [EntityRule](#entityrule) | |
 | destination | Destination match parameters. |  | [EntityRule](#entityrule) | |
 
+An `action` of `pass` will skip over the remaining Policy and jump to the
```

Comment:
I'm guessing this text is different because profiles are being deprecated but in that case I think we should still say that pass jumps to the profiles but that profiles are deprecated (until such point that we actually remove them).  Otherwise, it's misleading.